backend/app/crud/model_config.py

```
@@ -1,17 +1,25 @@
-import logging
+from datetime import datetime
 from typing import Any, Literal, get_args
+import logging
 
 from fastapi import HTTPException
+from sqlalchemy.exc import IntegrityError
 from sqlmodel import Session, select
 
 logger = logging.getLogger(__name__)
 
 from app.models import ModelConfig
```

Ruff (0.15.15)

[error] Module level import not at top of file

(E402)